openstack-helm/trilio-backup-targets/values_overrides/nfs.yaml

```diff
@@ -0,0 +1,17 @@
+trilio_backup_target:
+  backup_target_name: 'NFS_BackupTarget'
+  backup_target_type: 'nfs'
+  is_default: True
+  nfs_server: '10.10.0.1'
+  nfs_shares: /home/openstack-helm
+  nfs_options: "nolock,soft,timeo=600,intr,lookupcache=none,nfsvers=3,retrans=10"
+  storage_size: 20Gi
+  storage_class_name: nfs
+
+common:
+  vault_data_dir: /var/lib/trilio/triliovault-mounts
+  nova_user_id: "42424"
+  nova_group_id: "42424"
+
+images:
+  trilio_backup_targets: docker.io/trilio/trilio-wlm-helm:shruti-6.1.0-bobcat-2
```

Comment:
We need to use image nmae as trilio_wlm instead of trilio_backup_targets.
User should clearly know that is this wlm image.